pkg/mount/deleted_mount.go

```diff
@@ -25,7 +26,7 @@ func NewDeletedMounter(
 	mountImpl MountImpl,
 ) (*deletedMounter, error) {
 
-	devMounter, err := NewDeviceMounter([]string{"/dev/"}, mountImpl, nil, "")
+	devMounter, err := NewDeviceMounter([]*regexp.Regexp{regexp.MustCompile("/dev/")}, mountImpl, nil, "")
```

Comment:
when using as prefix, do we need to use this "^/dev/" instead?

Reply:
I did some testing in the go playground, if I just give it `"/dev/"` it will match `/dev/` anywhere in the string (prefix or not). I can't think of any other possibility where `/dev/` would show up so I think it's fine, but we can definitely add the caret if need be